php_thread.go

```diff
@@ -0,0 +1,55 @@
+package frankenphp
+
+import (
+	"net/http"
+	"runtime"
+)
+
+var phpThreads []*phpThread
+
+type phpThread struct {
+	threadIndex   int
+	mainRequest   *http.Request
+	workerRequest *http.Request
+	pinner        *runtime.Pinner
+	worker        *worker
+}
+
+func initializePHPThreads(numThreads int) {
+	phpThreads = make([]*phpThread, numThreads)
+	for i := 0; i < numThreads; i++ {
+		phpThreads[i] = &phpThread{threadIndex: i, pinner: &runtime.Pinner{}}
+	}
+}
+
+func getPHPThread(threadIndex int) *phpThread {
+	if threadIndex >= 0 && threadIndex < len(phpThreads) {
+		return phpThreads[threadIndex]
+	}
+	panic("no such thread")
+}
```

Comment:
```suggestion
	return phpThreads[threadIndex]
}
```

Nit but these checks are useless, this will already panic if the value is out of range. Also, using an uint will prevent passing negative numbers at compile time.

Reply:
Yeah you're right, I removed the redundant panics 👍